pkg/debuginfo/extract.go

```diff
@@ -142,8 +110,9 @@ var isDwarf = func(s *elf.Section) bool {
 
 var isSymbolTable = func(s *elf.Section) bool {
 	return s.Name == ".symtab" ||
-		s.Name == ".dynsymtab" ||
+		s.Name == ".dynsym" ||
 		s.Name == ".strtab" ||
+		s.Name == ".dynstr" ||
```

Comment:
Nice that these two symbol tables are added! I assume `dynsymtab` is not needed because they will be dynamic libraries and will be relocated? Not sure!

Reply:
We also to section type checks so if they're properly typed everything should work.

Also `.dynsymtab` is not a conventional section. I had made a mistake there previously.